web/themes/custom/server_theme/templates/server-theme-inner-element-contact-cta.html.twig

```diff
@@ -0,0 +1,29 @@
+{% set classes = [
+  'grid grid-cols-2 divide-x divide-gray-200 w-full',
+  align == 'start' ? 'items-start',
+  align == 'center' ? 'items-center',
+  align == 'end' ? 'items-end',
+] | join(' ') | trim %}
+<div class="{{ classes }}">
+  {% if email.url %}
+    <div>
+      <a href="{{ email.url }}" class="py-4 flex gap-x-4 justify-center text-gray-500 ">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true" class="w-5">
+          <path d="M3 4a2 2 0 00-2 2v1.161l8.441 4.221a1.25 1.25 0 001.118 0L19 7.162V6a2 2 0 00-2-2H3z"></path>
+          <path d="M19 8.839l-7.77 3.885a2.75 2.75 0 01-2.46 0L1 8.839V14a2 2 0 002 2h14a2 2 0 002-2V8.839z"></path>
+        </svg>
+        {{ email.text }}
+      </a>
+    </div>
+  {% endif %}
```

Comment:
This looks very similar to the phone. Maybe a chance to create own twig?

Also, on Drupal-starter we don't have nested values. But if you move to own twig, you could avoid that0